modules/integration_azure-datafactory/conf/03-adf-trigger-run.yaml

```diff
@@ -0,0 +1,23 @@
+module: "Azure DataFactory"
+name: "trigger error rate"
+filtering: "filter('resource_type', 'Microsoft.DataFactory/factories') and filter('primary_aggregation_type', 'true')"
+aggregation: ".sum(by=['name', 'azure_resource_group_name', 'azure_region'])"
+transformation: ".min(over='5m')"
+signals:
+  adf_trigger_succeeded_run:
+    metric: "triggerSucceededRuns"
+    rollup: rate
```

Comment:
while you make a percentage the default rollum sum will give the same result and it seems to be a most relevant value for troubleshooting